### PR TITLE
Set road_segment_id in news_flash of Suburban resolution.

### PR DESCRIPTION
**Pull request by ziv17**

### CODE REVIEW

**Comment by ziv17** [comment]
Hi @atalyaalon ,
This PR is the first part of issue #2567. It sets `road_segment_id` in news_flash items. After it is merged, we should update all news_flash items to set `road_segment_id` in them.
Note:
- Running the news flash update in my local environment, I saw that there are some news flash items that previously were considered not accident, and not that are considered as accidents. (e.g. id:173140).
- roads in NewsFlash where typed `float` in models.py, and I changed them to int.

File: anyway/models.py [hunk 1]
```diff
@@ -876,6 +876,7 @@ class NewsFlash(Base):
     street2_hebrew = Column(Text(), nullable=True)
     non_urban_intersection_hebrew = Column(Text(), nullable=True)
     road_segment_name = Column(Text(), nullable=True)
+    road_segment_id = Column(Integer(), nullable=True)
```
Comment (atalyaalon) [inline anyway/models.py:879]: @ziv17 we need an alembic version file fir this modification

File: anyway/widgets/widget_utils.py [hunk 2]
```diff
@@ -185,7 +185,7 @@ def get_involved_marker_view_location_filters(
         filters["street1_hebrew"] = location_info.get("street1_hebrew")
     elif resolution == BE_CONST.ResolutionCategories.SUBURBAN_ROAD:
         filters["road1"] = location_info.get("road1")
-        filters["road_segment_name"] = location_info.get("road_segment_name")
+        # filters["road_segment_name"] = location_info.get("road_segment_name")
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/widgets/widget_utils.py:188]: > @ziv17 I assume we want to delete this one?

Yes

File: anyway/models.py (hunk 1 above)
Comment (ziv17) [reply anyway/models.py:879]: This field is already in the table.

File: anyway/parsers/location_extraction.py [hunk 3]
```diff
@@ -534,6 +544,16 @@ def extract_geo_features(db, newsflash: NewsFlash, update_cbs_location_only: boo
         for resolution in all_resolutions:
             if resolution not in location_from_db:
                 setattr(newsflash, resolution, None)
+    if newsflash.road_segment_name is not None and newsflash.road1 is not None:
```
Comment (atalyaalon) [inline anyway/parsers/location_extraction.py:547]: @ziv17 perhaps add another condition that newsflash.road_segment_id is not None as well?

**Pull request merged**